core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -523,6 +523,8 @@ object Fiber extends FiberPlatformSpecific {
       def removeObserver(observer: Exit[E, A] => Unit)(implicit unsafe: Unsafe): Unit
 
       def poll(implicit unsafe: Unsafe): Option[Exit[E, A]]
+
+      def interrupt(cause: Cause[Nothing])(implicit unsafe: Unsafe): Unit
```

Comment:
Mima fails for this method missing from previous versions. I wouldn't have expected this could be a problem as the enclosing class is sealed + defined in the same module. Do I need to add `= ()`?